Java/jni/native_c4database.cc

```diff
@@ -82,7 +82,7 @@ Java_com_couchbase_litecore_C4Database_copy(JNIEnv *env, jclass clazz,
 /*
  * Class:     com_couchbase_litecore_C4Database
  * Method:    free
- * Signature: (J)
+ * Signature: (J)V
```

Comment:
Please fix the signature below to return void as well.

https://github.com/couchbase/couchbase-lite-core/blob/master/Java/src/com/couchbase/litecore/C4Database.java#L339